src/users/controller/edit-profile.js

```diff
@@ -1,26 +1,17 @@
-require("../../authentication/controller/passport-config");
-
 const UserModel = require("../../authentication/model/user-model");
 
 exports.updateProfile = async (req, res, next) => {
-  const obj = {};
-  const email = {};
-  email.email = req.userEmail;
-  const keys = ["firstname", "lastname", "phone"];
-
-  for (let itemsFromBodyIndex in req.body) {
```

Comment:
From the results of my tests,  the user is only allowed to edit the fields in the model. If the user enters something other than what is set in the model, it will not be registered.